keyboards/crkbd/keymaps/md40/readme.md

```diff
@@ -0,0 +1,23 @@
+# MD40: Minimally Disruptive 40% Keymap
+
+This keymap is an attempt to go from a standard keyboard layout to a 40% keyboard with minimal disruption.
+
+Some aims:
+- Two layers only
+- Existing keyboard positions where possible
+- Exceptions where it's not possible
+- Exceptions where it makes sense
+
+Supports crkbd only.
+
+## Base Layer
+
+![Base Layer](https://raw.githubusercontent.com/shano/shano.github.io/main/images/base.png)
+
+
+
+
+
+## Fn Layer
+
+![Base Layer](https://raw.githubusercontent.com/shano/shano.github.io/main/images/fn.png)
```

Comment:
These images should be rehosted on Imgur.

Reply:
Moved to imgur.